pkgs/development/tools/cambalache/default.nix

```diff
@@ -40,6 +40,8 @@ python3.pkgs.buildPythonApplication rec {
     desktop-file-utils # for update-desktop-database
     shared-mime-info # for update-mime-database
     wrapGAppsHook
+    gtk4 # checks for broadwayd and gtk4-broadwayd during configure
```

Comment:
Those are used at runtime so I would expect them to be found from buildInputs.

Reply:
patched